### DISCUSSION TITLE
Rules for package names

### DISCUSSION THREAD

**Discussion by rchastain** [mse-org/mseide-msegui#56] [General]
Hello Fred!

I am thinking of writing a tutorial about MSEide installation, and maybe a script (at least for Linux).

There is a little thing that I noticed while installing v5.6.10.

For example, in _mseide_5_6_10_linux_amd64.zip_, there is a folder named _mseide_lin64_. Shouldn't it be _bin_? This is how Martin used to do, and it's simpler. (Of course, you are the master, and maybe you have a good reason to do it that way, but I wanted to say my opinion.)

I also notice that you changed the way to name the ZIP file itself. For example, _mseide_5_6_10_linux_amd64.zip_ instead of _mseide_linux_amd64_5_6_10.zip_. This isn't very important, but why not keep the same order?

A last thing. To get MSEgui, we have to click on "Source code (zip)", isn't it? And the file is named _mseide-msegui-5.6.10.zip_ instead of  _mseide_msegui_src_5_6_10.zip_. So I would suggest to add a ZIP named like that in future releases.

My two cents.

Best regards dear Fred.

Roland

**Comment by fredvs** [comment]
>For example, in mseide_5_6_10_linux_amd64.zip, there is a folder named mseide_lin64. Shouldn't it be bin? This is how Martin used to do, and it's simpler. 

Hum, it is because you may install the 32 bit or/and 64 bit version on a mullti-arch system so different name for eaach folder is the easy way.
I have both  mseide_lin64 and mseide_lin32 installed on my Debian 11 multi arch.
The same for Windows, mseide_win64 and mseide_win32 are installed on my Windows 10 system.
But if you find it too complicated, of course we will change it.

> (Of course, you are the master, ....

Ho, non, please, I am not the master and never want to be, maybe a servant but not a master...

> I also notice that you changed the way to name the ZIP file itself. For example, mseide_5_6_10_linux_amd64.zip instead of mseide_linux_amd64_5_6_10.zip. This isn't very important, but why not keep the same order?

Yes, of course, what order do you prefer?

> A last thing. To get MSEgui, we have to click on "Source code (zip)", isn't it? And the file is named mseide-msegui-5.6.10.zip instead of mseide_msegui_src_5_6_10.zip. So I would suggest to add a ZIP named like that in future releases.

Yes, good idea.


Fre;D
